### PR TITLE
Ping command now shows ping, issue #156

### PR DESCRIPTION
**Pull request by albertopoljak**

### CODE REVIEW

**Comment by meooow25** [comment]
Not exactly sure this is what we want. This would be the time taken to send an HTTP request to Discord API. Will investigate and update here tomorrow.

**Comment by meooow25** [comment]
This change is alright, if you could also add the websocket/gateway latency the PR is good to go. This can be obtained as `bot.latency`. The message should look like:

**REST API latency**: 345ms
**Gateway API latency**: 678ms

File: tle/cogs/meta.py [hunk 1]
```diff
@@ -69,7 +70,12 @@ def __init__(self, bot):
     @meta.command(brief='Is TLE up?')
     async def ping(self, ctx):
         """Replies to a ping."""
-        await ctx.send("pong")
+        start = time.perf_counter()
+        message = await ctx.send(':ping_pong: Pong!')
+        end = time.perf_counter()
+        duration = (end - start) * 1000
+        await message.edit(content=f'REST API latency: {int(duration)}ms \n'
```
Comment (krofna) [inline tle/cogs/meta.py]: The extra whitespace before the newline is slightly frustrating.

Comment (albertopoljak) [reply tle/cogs/meta.py]: Ayy sorry. 
Little design issues like this are hardest.
At first I had whitespace but `{int(duration)}ms\n` just seemed like too much special characters without a break so I added whitespace for easier reading I guess.
Reverted back now.

File: tle/cogs/meta.py [hunk 2]
```diff
@@ -69,7 +70,12 @@ def __init__(self, bot):
     @meta.command(brief='Is TLE up?')
     async def ping(self, ctx):
         """Replies to a ping."""
-        await ctx.send("pong")
+        start = time.perf_counter()
+        message = await ctx.send(':ping_pong: Pong!')
```
Comment (meooow25) [inline tle/cogs/meta.py]: `:ping_pong:` seems unnecessary. Also it is removed in the edit anyway.

**Comment by krofna** [comment]
I'm merging this. Thank you for your contribution.

**Pull request merged**